opengrok-tools/src/main/python/opengrok_tools/utils/command.py

```diff
@@ -333,6 +333,7 @@ def fill_arg(self, args_append=None, args_subst=None):
         newcmd = []
         subst_done = -1
         for i, cmdarg in enumerate(self.cmd):
+            cmdarg = str(cmdarg)
```

Comment:
There are other places where items in self.cmd are assumed to be strings. For example `Command.__str__()`
I'd suggest to do this assignment in `Command.__init__()` instead:
```
self.cmd = map(str, self.cmd)
```

Reply:
will be done in #3225 , thanks for the neat idea !